pkg/apmintegration/file.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package apmintegration
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ava-labs/apm/types"
+	"github.com/ava-labs/avalanche-cli/pkg/application"
+	"gopkg.in/yaml.v3"
+)
+
+func GetRepos(app *application.Avalanche) ([]string, error) {
+	repositoryDir := filepath.Join(app.ApmDir, "repositories")
+	orgs, err := os.ReadDir(repositoryDir)
+	if err != nil {
+		return []string{}, err
+	}
+
+	output := []string{}
+
+	for _, org := range orgs {
+		repoDir := filepath.Join(repositoryDir, org.Name())
+		repos, err := os.ReadDir(repoDir)
+		if err != nil {
+			return []string{}, err
+		}
+		for _, repo := range repos {
+			output = append(output, org.Name()+"/"+repo.Name())
+		}
+	}
+
+	return output, nil
+}
+
+func GetSubnets(app *application.Avalanche, repoAlias string) ([]string, error) {
+	subnetDir := filepath.Join(app.ApmDir, "repositories", repoAlias, "subnets")
+	subnets, err := os.ReadDir(subnetDir)
+	if err != nil {
+		return []string{}, err
+	}
+	subnetOptions := make([]string, len(subnets))
+	for i, subnet := range subnets {
+		subnetOptions[i] = subnet.Name()[:len(subnet.Name())-5]
+	}
+
+	return subnetOptions, nil
+}
+
+type SubnetWrapper struct {
+	Subnet types.Subnet `yaml:"subnet"`
+}
+
+type VMWrapper struct {
+	VM types.VM `yaml:"vm"`
+}
```

Comment:
if I understand correctly these weird wrapper things are needed because of the funny key in the apm definitions (e.g vm/subnet). I think we can probably just remove these from the definition spec (I can do this) to make this cleaner and make it so we don't need these wrappers

Reply:
Yes, I think that would be a good fix